### PR TITLE
chore: cleanup incorrect apostrophe

### PR DESCRIPTION
**Pull request by lucemans**
## Changes

Well I mean, as per #2157 , and #2142 , apostrophe.

Don't you love those improved apostrophe's?

### CODE REVIEW

File: docs/src/pages/es/reference/api-reference.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ export async function getStaticPaths({rss}) {
   // Generate an RSS feed from this collection
   rss({
     // The RSS Feed title, description, and custom metadata.
-    title: 'Don’s Blog',
+    title: 'Don's Blog',
```
Comment (FredKSchott) [inline docs/src/pages/es/reference/api-reference.md:225]: 💀

File: scripts/notify/index.js [hunk 2]
```diff
@@ -14,10 +14,10 @@ const verbs = [
 	'hit the registry!',
 	'to share!',
 	'for you!',
-	'for y’all! 🤠',
-	'comin’ your way!',
-	'comin’ atcha!',
-	'comin’ in hot!',
+	'for y'all! 🤠',
```
Comment (FredKSchott) [inline scripts/notify/index.js:17]: 💀

File: examples/blog-multiple-authors/src/pages/posts/[...page].astro [hunk 3]
```diff
@@ -28,7 +28,7 @@ export async function getStaticPaths({ paginate, rss }) {
 }
 
 // page
-let title = 'Don’s Blog';
+let title = 'Don's Blog';
```
Comment (FredKSchott) [inline examples/blog-multiple-authors/src/pages/posts/[...page].astro:31]: 🗡️

File: examples/blog-multiple-authors/src/pages/posts/[...page].astro [hunk 4]
```diff
@@ -12,7 +12,7 @@ export async function getStaticPaths({ paginate, rss }) {
 	// Generate an RSS feed from this collection of posts.
 	// NOTE: This is disabled by default, since it requires `buildOptions.site` to be set in your "astro.config.mjs" file.
 	//   rss({
-	//     title: 'Don’s Blog',
+	//     title: 'Don's Blog',
```
Comment (FredKSchott) [inline examples/blog-multiple-authors/src/pages/posts/[...page].astro:15]: 🗡️

File: docs/src/pages/es/guides/rss.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ export async function getStaticPaths({rss}) {
   // Genera un feed RSS de esta colección
   rss({
     // El título, la descripción y los metadatos personalizados de la fuente RSS.
-    title: 'Don’s Blog',
+    title: 'Don's Blog',
```
Comment (FredKSchott) [inline docs/src/pages/es/guides/rss.md:24]: ☠️

File: scripts/notify/index.js (hunk 2 above)
Comment (svemat01) [reply scripts/notify/index.js:17]: It appears you forgot to escape one of the quotes.

**Comment by FredKSchott** [comment]
I still love this but can't merge with bugs!

**Pull request closed without merging**